test.js

```diff
@@ -1,51 +1,100 @@
 const net = require('net');
 const test = require('ava');
-const pify = require('pify');
 
 const { findFreePort, isPortFree } = require('.');
 
-test('should return the passed port value', async (t) => {
+const requestedPort = 3010;
+const requestedHost = '127.0.0.1';
+const defaultHost = '0.0.0.0';
+
+const wrapper = (t, run) => new Promise(resolve => run(t, resolve));
+
+test.serial('should return the passed port value', async (t) => {
   const port = await findFreePort(3000);
 
   t.is(port, 3000);
 });
 
-test('should throw a port searching error when finding port outside the available range', async (t) => {
+test.serial('should throw a port searching error when finding port outside the available range', async (t) => {
   const port = await t.throwsAsync(() => findFreePort(65536));
 
   t.true(port.message.includes('should be >= 0 and < 65536. Received 65536.'));
 });
 
-test('should return `true` when available port is chosen', async (t) => {
+test.serial('should return `true` when available port is chosen', async (t) => {
```

Comment:
I think we can simplify phrases here and everywhere to remove passive voice.

> should return `true` when port is available

Or even:

> should return `true` when port is free

Reply:
Simplified tests' title.